### PR TITLE
Handful of copyedits to README.md

### PR DESCRIPTION
**Pull request by wmpea**
A few copyedits for improved grammatical polish and clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ Participation in the Zcash project is subject to a
...
 
-Build Zcash along with most dependencies from source by running:
+Build Zcash along with most dependencies from source by running the following command:
 
 ```
 ./zcutil/build.sh -j$(nproc)
 ```
 
-Currently only Linux is officially supported.
+Currently, Zcash is only built for Linux.
```
Comment (garethtdavies) [inline README.md]: I think the original is more accurate for this line as it builds on multiple platforms/architectures but only Linux is officially supported?

Comment (wmpea) [reply README.md]: Sounds good! Ty!

Comment (str4d) [reply README.md]: I agree. Please make this change, or something along the lines of:

```suggestion
Currently, Zcash is only officially supported on Debian and Ubuntu.
```

(swapping out `Linux` for [greater accuracy](https://ci.z.cash/#/builders?tags=%2Bsupported)).

**Comment by Eirik0** [comment]
@zkbot r+

**Pull request merged**